app/src/main/res/values/strings.xml

```diff
@@ -22,8 +22,6 @@
     <string name="no_private_tabs_description">Your private tabs will be shown here.</string>
     <!-- Message announced to the user when tab tray is selected with 1 tab -->
     <string name="open_tab_tray_single" moz:removedIn="96" tools:ignore="UnusedResources">1 open tab. Tap to switch tabs.</string>
-    <!-- Message announced to the user when tab tray is selected with 0 or 2+ tabs -->
-    <string name="open_tab_tray_plural" moz:removedIn="95" tools:ignore="UnusedResources">%1$s open tabs. Tap to switch tabs.</string>
```

Comment:
Forgot the process here, do the translations need to be cleaned up too?

Reply:
I only started doing this last December. I documented it here https://github.com/mozilla-mobile/fenix/wiki/Removing-strings#when-to-remove-an-unused-string-and-how. Since we no longer ship Firefox 95 (since Release is now 96) starting today, we can safely cleanup strings marked with `moz:removedIn="95"`. There's no formal process that says we have to do it immediately, but I like to clean things up.